modules/rostests/apitests/kernel32/InitOnce.c

```diff
@@ -154,6 +154,7 @@ START_TEST(InitOnce)
 
     /* Re-initialize One-Time initialization structure by using INIT_ONCE_STATIC_INIT */
     InitOnce = (INIT_ONCE)INIT_ONCE_STATIC_INIT;
+    ulContextData = 0xdeadbeef;
```

Comment:
What about using `0xdeadbeefdeadbeefULL` like done in other apitests with x64 support?

Reply:
it doesn't really matter.